x/ccv/consumer/keeper/distribution.go

```diff
@@ -153,7 +152,7 @@ func (k Keeper) SendRewardsToProvider(ctx sdk.Context) error {
 				sdk.NewAttribute(ccv.AttributeDistributionCurrentHeight, strconv.Itoa(int(currentHeight))),
 				sdk.NewAttribute(ccv.AttributeDistributionNextHeight, strconv.Itoa(int(currentHeight+k.GetBlocksPerDistributionTransmission(ctx)))),
 				sdk.NewAttribute(ccv.AttributeDistributionFraction, (k.GetConsumerRedistributionFrac(ctx))),
-				sdk.NewAttribute(ccv.AttributeDistributionTotal, fpTokens.String()),
+				sdk.NewAttribute(ccv.AttributeDistributionTotal, allBalances.String()),
```

Comment:
would something like `AttributeTotalFeePool` be a better name here? Likely not important tho

Reply:
i can change it here, it is a var name, so not api breaking right?

Reply:
I'm not sure what API breaking would mean in this context, if you change the value string then the event will change (which seems ok? I don't think changing an event is state breaking but I could be wrong), if you just change the var naming then it's a pure refactor. Again not super important, so feel free to ignore my comment if we're not confident how the change would affect things